packages/core/test/toast/overlayToasterTests.tsx

```diff
@@ -174,9 +174,21 @@ describe("OverlayToaster", () => {
                 toaster.show(toast);
                 assert.isFalse(errorSpy.calledWithMatch("two children with the same key"), "mutation side effect!");
             });
+        });
 
-            it("does not exceed the maximum toast limit set", () => {
+        describe("with maxToasts set to finite value", () => {
+            before(() => {
+                testsContainerElement = document.createElement("div");
+                document.documentElement.appendChild(testsContainerElement);
                 toaster = spec.create({ maxToasts: 3 }, testsContainerElement);
+            });
+
+            after(() => {
```

Comment:
Without the additional cleanup in a few tests that call `OverlayToaster.create()` directly (instead of using the existing `Toaster` instance), I was seeing the "_does not attach toast container to body on script load_" test fail.